content/cloud-docs/api-docs/teams.mdx

```diff
@@ -143,6 +144,7 @@ Properties without a default value are required.
 | --------------------------------------- | ------ | ---------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | `data.type`                             | string |            | Must be `"teams"`.                                                                                                                                                                                                                   |
 | `data.attributes.name`                  | string |            | The name of the team, which can only include letters, numbers, `-`, and `_`. This will be used as an identifier and must be unique in the organization.                                                                              |
+| `data.attributes.sso-team-id`           | string | (nothing)  | The unique identifier of the team referred to from the SAML MemberOf attribute.                                                                                                                                                      |
```

Comment:
This field is optional and is only usable if the team that is being created/updated belongs to an organization that is in the business tier.